méli-mélo/2023-09-distance-calculator/index.html

```diff
@@ -0,0 +1,309 @@
+</label>
+</li>
+<li class="checkbox">
+<label>
+<input type="checkbox" name="location" class="wb-tagfilter-ctrl" value="location__ottawa"> Ottawa
+</label>
+</li>
+<li class="checkbox">
+<label>
+<input type="checkbox" name="location" class="wb-tagfilter-ctrl" value="location__toronto"> Toronto
+</label>
+</li>
+<li class="checkbox">
+<label>
+<input type="checkbox" name="location" class="wb-tagfilter-ctrl" value="location__vancouver"> Vancouver
+</label>
+</li>
+</ul>
+</fieldset>
+</div>
+<div class="form-group mrgn-bttm-0">
+<label for="dayList">Day of the week</label>
+<select id="dayList" name="dayOTW" class="form-control full-width wb-tagfilter-ctrl">
+<option value="">All</option>
+<option value="dayOTW_monday">Monday</option>
+<option value="dayOTW_tuesday">Tuesday</option>
+<option value="dayOTW_wednesday">Wednedsday</option>
+<option value="dayOTW_thursday">Thursday</option>
+<option value="dayOTW_friday">Friday</option>
+<option value="dayOTW_saturday">Saturday</option>
+<option value="dayOTW_sunday">Sunday</option>
+</select>
+</div>
+</details>
+</div>
+<div class="col-xs-12 col-sm-8 col-lg-9">
+<ul class="list-unstyled events-list wb-tagfilter-items">
+
+<li data-wb-tags="language__english language__french location__online dayOTW_friday">
+<h3>Ask PAC Anything!</h3>
+<p><span class="event-date">Friday August 5th, 2022</span> | <span class="event-time">12:00 PM to 1:00 PM</span></p>
+<p><strong>Location:</strong> <span class="event-location">Online</span>, <strong>Language</strong>: <span class="event-language">English, French</span></p>
+</li>
+
+<li data-wb-tags="language__english location__vancouver dayOTW_wednesday">
+<h3>Bidding on Opportunities</h3>
+<p><span class="event-date">Wednesday August 10th, 2022</span> | <span class="event-time">2:00 PM to 3:30 PM</span></p>
+<p><strong>Location:</strong> <span class="event-location">Vancouver</span>, <strong>Language</strong>: <span class="event-language">English</span></p>
+<div class="distance-visible hidden"><p><strong><span class="distance" data-gc-distance-coordinates='{ "longtitude":"-123.113889","latitude":"49.261111" }'></span> Km</strong> from <span class="distance-title"></span></p></div>
+<div class="distance-sort hidden"></div>
+</li>
+
+<li data-wb-tags="language__english location__ottawa dayOTW_thursday">
+<h3>Doing business with the Government of Canada (Webinar)</h3>
+<p><span class="event-date">Thursday August 11th, 2022</span> | <span class="event-time">12:00 PM to 2:00 PM</span></p>
+<p><strong>Location:</strong> <span class="event-location">Ottawa</span>, <strong>Language</strong>: <span class="event-language">English</span></p>
+<div class="distance-visible hidden"><p><strong><span class="distance" data-gc-distance-coordinates='{ "longtitude":"-75.58429","latitude":"45.33339" }'></span> Km</strong> from <span class="distance-title"></span></p></div>
+<div class="distance-sort hidden"></div>
```

Comment:
Considering those element is updated and do show status. Adaptation might (more likely) is required for a conformance with the [SC 4.1.3 Status Message](https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html)

Reply:
Added simple status message to example in 9b57565